src/main/java/io/appium/java_client/android/AndroidMobileOptions.java

```diff
@@ -0,0 +1,1733 @@
+        Object duration = getCapability(AndroidMobileCapabilityType.ANDROID_INSTALL_TIMEOUT);
+        if (duration.getClass().isAssignableFrom(Long.class)) {
+            return Duration.ofMillis((Long) duration);
+        }
+        Integer integer = (Integer) duration;
+        Long value = Long.valueOf(integer);
+        return Duration.ofMillis(value);
+    }
+
+    /**
+     * Set the app to allow for correct handling of orientation on landscape-oriented devices.
+     *
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#ANDROID_NATURAL_ORIENTATION
+     */
+    public AndroidMobileOptions setAndroidNaturalOrientation() {
+        return setAndroidNaturalOrientation(true);
+    }
+
+    /**
+     * Set whether to allow for correct handling of orientation on landscape-oriented devices.
+     *
+     * @param bool when true allows for correct handling of orientation on landscape-oriented devices.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#ANDROID_NATURAL_ORIENTATION
+     */
+    public AndroidMobileOptions setAndroidNaturalOrientation(boolean bool) {
+        return amend(AndroidMobileCapabilityType.ANDROID_NATURAL_ORIENTATION, bool);
+    }
+
+    /**
+     * Get whether correct handling of orientation on landscape-oriented devices is allowed.
+     *
+     * @return true if correct handling of orientation on landscape-oriented devices is allowed.
+     * @see AndroidMobileCapabilityType#ANDROID_NATURAL_ORIENTATION
+     */
+    public boolean isAndroidNaturalOrientation() {
+        return (boolean) getCapability(AndroidMobileCapabilityType.ANDROID_NATURAL_ORIENTATION);
+    }
+
+    /**
+     * Set the name of the directory on the device in which the screenshot will be put.
+     *
+     * @param path is the directory on the device in which the screenshot will be put.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#ANDROID_SCREENSHOT_PATH
+     */
+    public AndroidMobileOptions setAndroidScreenshotPath(String path) {
+        return amend(AndroidMobileCapabilityType.ANDROID_SCREENSHOT_PATH, path);
+    }
+
+    /**
+     * Get the name of the directory on the device in which the screenshot will be put.
+     *
+     * @return String of the name of the directory on the device in which the screenshot will be put.
+     * @see AndroidMobileCapabilityType#ANDROID_SCREENSHOT_PATH
+     */
+    public String getAndroidScreenshotPath() {
+        return (String) getCapability(AndroidMobileCapabilityType.ANDROID_SCREENSHOT_PATH);
+    }
+
+    /**
+     * Set the activity name for the Android activity you want to launch from your package.
+     *
+     * @param activity is the name of the activity to launch from the package.
+     * @return this MobileOptions, for chaining.
+     * @see AndroidMobileCapabilityType#APP_ACTIVITY
+     */
+    public AndroidMobileOptions setAppActivity(String activity) {
```

Comment:
I think it makes sense to add a link to the documentation, because users quite often have difficulties understanding what activity name concept is: https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/android/activity-startup.md